app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   include RivalUser
   include TwitchUser
 
-  has_many :runs
+  has_many :runs, dependent: :destroy
```

Comment:
We give users the option to keep their runs when deleting: 

<img width="839" alt="screenshot 2018-06-19 10 02 06" src="https://user-images.githubusercontent.com/438911/41612459-dad3d9ea-73a7-11e8-9db6-ae99e8f29e6f.png">

The [code that handles this][1] does nothing to the runs if the user chooses to anonymize them, which seems wrong now that I'm looking at it. So should probably be

- `:nullify` if we want to keep that code as-is, or
- `:restrict_with_exception` if we want to fix that code to disown all runs, since in that situation, trying to destroy a user with any runs would be a bug since they all should have already been destroyed or disowned.

[1]: https://github.com/glacials/splits-io/blob/master/app/controllers/users_controller.rb#L8-L13

Reply:
I'll revert this one, and make an issue for this.  It should probably be restricted with exception though since nullify doesn't make sense if the user wants all runs deleted, so the code that handles this should raise an error if some are left.